FunctionSignatureMap.php

```diff
@@ -4686,15 +4688,15 @@
 'http\Params::toString' => ['string'],
 'http\QueryString::__construct' => ['void', 'querystring'=>'string'],
 'http\QueryString::__toString' => ['string'],
-'http\QueryString::get' => ['http\QueryString|mixed|string', 'name='=>'string', 'type='=>'mixed', 'defval='=>'mixed', 'delete='=>'bool'],
-'http\QueryString::getArray' => ['array', 'name'=>'string', 'defval='=>'mixed', 'delete='=>'bool'],
-'http\QueryString::getBool' => ['bool', 'name'=>'string', 'defval='=>'mixed', 'delete='=>'bool'],
-'http\QueryString::getFloat' => ['float', 'name'=>'string', 'defval='=>'mixed', 'delete='=>'bool'],
+'http\QueryString::get' => ['http\QueryString|string|mixed', 'name='=>'string', 'type='=>'mixed', 'defval='=>'mixed', 'delete='=>'bool|false'],
+'http\QueryString::getArray' => ['array|mixed', 'name'=>'string', 'defval='=>'mixed', 'delete='=>'bool|false'],
```

Comment:
Oh. It returns $defval, which can be anything.

Nit: `bool|false` seems unnecessary - is there a reason psalm also passes false?